### PR TITLE
New release start page.

### PR DESCRIPTION
**Pull request by ryanleesipes**
New start page for release.

### CODE REVIEW

**Comment by Sancus** [comment]
This looks ok. My biggest concern is that the entire top part of the page right now is a donation request, essentially. That's no longer the case with this page, and particularly on lower resolution screens and when the window is shrunk, the "Get Support" thing ends up on top and it is possible the donation section could be completely invisible.

This would normally be relatively minor, but given that this page is the source of almost all our revenue, even relatively small visibility changes could have a significant impact.

**Comment by Sancus** [comment]
New Page:
image
image

Old Page:
image
image

**Comment by mkmelin** [comment]
For the first sentence in "Get Support", question mark last? (Need help with something in Thunderbird**?**)

I'd perhaps also replace the "Please take the time... " sentence or remove it as it doesn't add much. 

Something like the old "If you like Thunderbird, please consider a donation", bolded should be more prominent somewhere in the page, maybe instead of "Please take the time... ". For the absolute majority, that's the only way they would and could contribute.

The old text was also written to include a balanced sense of urgency to donate, which really still applies.  The new text is more of "donate if you want to make us happy". Can you add more urgency to it?

**Comment by ryanleesipes** [comment]
I am working on this. I really dislike how things get laid out. So I'm taking a look at the CSS to see if I can make some adjustments. In my opinion, the sections should always show from the top, left -> right, next row left -> right.

So I'm trying to make that happen.

**Comment by ryanleesipes** [comment]
I've made some adjustments. I'm still trying to understand the CSS/less that governs this page.

Single column

single-column-release-start-page

Multi-column

new-updated-start-page

**Comment by ryanleesipes** [comment]
In the short-term, I think this is probably sufficient. But I would like to collaborate with Kohei and perhaps the Ura Design guys beautify this page even more in the future.

File: start-page/release/index.html [hunk 1]
```diff
@@ -10,26 +10,37 @@
...
       </header>
       <div>
         <section>
-          <h2><i class="far fa-life-ring fa-lg" style="color:#0060df"></i> {{ _('<a href="%(support)s">Get Support</a>')|format(support='https://support.mozilla.org/products/thunderbird') }}</h2>
-          <p>{{ _('Need help with something in Thunderbird. We have a <a href="%(support)s">support page</a> where you ask questions and read knowledge base articles to learn how to use Thunderbird.')|format(support='https://support.mozilla.org/products/thunderbird') }}</p>
+          <h2>{{ _('Why we need donations') }}</h2>
+              <p>
+              {{ _('<strong>Thunderbird improvements are no longer paid for by Mozilla.</strong> Fortunately there is an active community keeping it running and developing it further. But to survive long term, the project needs funding.') }}
```
Comment (mkmelin) [inline start-page/release/index.html]: Maybe not bold this. And perhaps phrase it as "Thunderbird is no longer funded by Mozilla"

File: start-page/release/index.html [hunk 2]
```diff
@@ -10,26 +10,37 @@
...
+              </p>
+              <p>
+              {{ _('Thunderbird is currently transitioning to an independent organization. Being independent, we can shape our own fate, but there is significant infrastructure that must be maintained to deliver the application to our tens of millions of users. For Thunderbird to survive and continue to evolve, we need your support and ask for your donation today. All the money donated will go directly to funding Thunderbird development and infrastructure.') }}
```
Comment (mkmelin) [inline start-page/release/index.html]: This actually relates to the time we we're seeking a new financial home. But we got that within MoFo in the end. Should take the time to adjust this text.

File: start-page/release/index.html [hunk 3]
```diff
@@ -10,26 +10,37 @@
...
-          <h2><i class="fas fa-bug fa-lg" style="color:#058b00"></i> {{ _('<a href="%(bug)s">Report a Bug</a>')|format(bug='https://bugzilla.mozilla.org/describecomponents.cgi?product=Thunderbird') }}</h2>
-          <p>{{ _('Think you\'ve found a problem with Thunderbird? <a href="%(bug)s">Create a bug report</a> so that we can make sure it gets solved. Helping us identify and fix bugs makes for a better Thunderbird for all.')|format(bug='https://bugzilla.mozilla.org/describecomponents.cgi?product=Thunderbird') }}</p>
+          <h2><i class="fas fa-heart fa-lg"  style="color:#ff0039"></i> {{ _('<a href="%(donate)s">Donate to Thunderbird</a>')|format(donate='https://donate.mozilla.org/en-US/thunderbird/?presets=100,50,25,15&amount=50&utm_source=start_page_thunderbird&utm_medium=referral&utm_content=header&currency=usd') }}</h2>
+            <p>{{ _('<strong>Thunderbird is completely funded by donations!</strong> Thunderbird is the leading open source, cross-platform and calendaring client, free for business and personal use. We want it to stay secure and become even better. A donation will allow us to hire developers, pay for infrastructure, and continue to improve.') }}</p>
```
Comment (mkmelin) [inline start-page/release/index.html]: "Completely" is almost true, but there are minor income streams and those might grow, so I think this needs adjustment as well.

Comment (mkmelin) [reply start-page/release/index.html]: On a general note, it's more effectful with the boldings if they are not the first thing after the header.

**Comment by wsmwk** [comment]
On 6/14/2018 4:39 PM, Magnus Melin wrote:
>
> "Completely" is almost true, but there are minor income streams and 
> those might grow, so I think this needs adjustment as well.
>
I would simply drop the word - reads fine without it.

File: start-page/release/index.html (hunk 1 above)
Comment (ryanleesipes) [reply start-page/release/index.html]: I will adjust this today.

File: start-page/release/index.html (hunk 2 above)
Comment (ryanleesipes) [reply start-page/release/index.html]: I addressed this, take a look at it now.

File: start-page/release/index.html (hunk 3 above)
Comment (ryanleesipes) [reply start-page/release/index.html]: Let me know what you think of the changes I made, used bold in different places.

**Comment by wsmwk** [comment]
I think "completely" should be removed or replaced, because it 
diminishes the investment Mozilla continues to make, plus those of our 
other eventual funding streams, partners, etc

**Comment by ryanleesipes** [comment]
Completely has been removed.

**Comment by ryanleesipes** [comment]
Other changes were made as well in https://github.com/thundernest/thunderbird-website/pull/41/commits/22803adcab0a45dd9ac293f479d1bcb4e859b1a3 addressing Magnus' concerns. @mkmelin please review.

**Comment by ryanleesipes** [comment]
@mkmelin sent another commit: https://github.com/thundernest/thunderbird-website/pull/41/commits/7a8377319d0b99d081d126544db8c37da4cd4558

**Comment by ryanleesipes** [comment]
Any further thoughts on this?

**Comment by mkmelin** [comment]
Not yet, I'll check it out in the next days.

**Comment by ryanleesipes** [comment]
Updated screenshots below! @wsmwk @mkmelin 

full-size

column

**Comment by wsmwk** [comment]
A bit of bikeshedding from a conversation on IRC yesterday.  I have two concerns:

1. Get Involved section - our revisions have reduced the focus on developers/developing code, so as to not scare off non-developers, because we very much need more non-developer contributors. But the last sentence "Become part of the team that helps create Thunderbird", which has a great ring, still, to me signals "developers".  I don't have a better wording at hand but I will try to develop something.  

2. Report a bug is the last paragraph. I'd like to have the flexibility to entirely hide that section if needed (for example if we start getting too many reports  via start page that are reallly support requests - as noted in IRC, Firefox does not have "report a bug" in release channel UI for a reason).  Sancus said this is easy with the CSS.  I envision having reporting enabled at least in the early days of 60.0 and 60.1.0.  Conversely, removing reporting gives greater emphasis to the remaining three items in the users' eyes,  which is a good thing.

**Comment by wsmwk** [comment]
On 7/16/2018 4:25 PM, Ryan Sipes wrote:
>
> @wsmwk <https://github.com/wsmwk> I'll just remove the report bug 
> point entirely.
>
But note, I also said I am interested it staying there for initial 
release periods.  (It might be a mistake, but I'd like to test it out)


> As for the get involved section, I think that something along the 
> lines of "Become part of the team that makes Thunderbird great" <- Is 
> that closer to what you want?
>
How about "Become part of the team that supports Thunderbird users, and 
makes Thunderbird great"

**Comment by kewisch** [comment]
For me personally the phrase of making something great (again) is burned for life. Can we use different wording?

**Comment by ryanleesipes** [comment]
Agreed, maybe: "Become part of a team that supports Thunderbird users, and makes Thunderbird awesome."

**Comment by ryanleesipes** [comment]
This has been changed to match the above wording ^

**Comment by Sancus** [comment]
This looks good now. One comment is that I feel like we should revisit the use of so many fonts on this page... we might as well just use the fontawesome fonts and get rid of all the old opensans stuff. Over half of the page size is font files now, which seems more than a little ridiculous.

I'm still going to r+ and merge this because I want l10n to get started, though, so font changes will have to be in a follow up, or I'll do them at some point when I have time.

**Comment by Sancus** [comment]
Live here: https://start-stage.thunderbird.net/en-US/release/

Will stay on stage for a week or two to allow for l10n, I'll monitor how many languages and whether we have all the major ones. If we have the big ones(de, fr, it, ja, es-ES, pl, ru) we can go live at the same time as TB60 ESR.

**Comment by mkmelin** [comment]
The old page had a bolded "If you like Thunderbird, please consider a donation!" - somehow I think this is a message that has touched people.

Now there is "Thunderbird is funded by donations! " - but that does not make it clear to people that it's *their* donations that are needed. 

I suggest adding back the old bolded text after the "Thunderbird is funded by donations", or somehow make sure it's very clear from whom donations are expected. Users may easily think it's corporate donations.

Sorry for being late to the party

**Comment by Sancus** [comment]
Can make another PR to change things. Please do so quickly though, as a string change will reset localization efforts that have already started, and churning this too much can justifiably upset localizers.

**Pull request merged**